src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -49,7 +50,12 @@ AdaptDispatch::AdaptDispatch(std::unique_ptr<ConGetSet> pConApi,
 // - <none>
 void AdaptDispatch::Print(const wchar_t wchPrintable)
 {
-    _pDefaults->Print(_termOutput.TranslateKey(wchPrintable));
+    const auto wchTranslated = _termOutput.TranslateKey(wchPrintable);
+    // The DEL character will only be ouput if translated to something else.
```

Comment:
output*

Reply:
Thanks for catching that, I'll fix it next time I rebase.